app/models/item.rb

```diff
@@ -12,6 +12,13 @@ class Item <ApplicationRecord
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
 
+  def self.most_popular_five
+    Item.joins(:item_orders).order(quantity: :desc).limit(5)
+  end
+
+  def self.least_popular_five
+    Item.joins(:item_orders).order(:quantity).limit(5)
+  end
```

Comment:
I'm not sure these methods are accomplishing what we want them to. If for example there are multiple item orders for the same item, then the most_popular_five method will list each item order from highest quantity to lowest quantity, producing different results for the same item. In my mind, we need to group the item orders by item, sum the quantity, and then order the items based on descending total quantity.

Reply:
I see what you're saying. I'll take a look at that!

Reply:
Fixed!